server/controllers/users/updatePassword.js

```diff
@@ -0,0 +1,23 @@
+const Joi = require('@hapi/joi');
+
+const hashPassword = require('./helpers/hashPassword');
+const updatePassword = require('../../database/queries/updatePassword');
+
+const schema = Joi.object().keys({
+  password: Joi.string().min(6),
+});
+
+exports.postPassword = (req, res, next) => {
```

Comment:
I think it'd be better if you named it `updatePasswork`, and the one on line 4 to `updatePassQuery`